packages/site-docs/src/styles/_examples.scss

```diff
@@ -199,6 +199,12 @@
   }
 }
 
+.docs-tooltip2-example {
+  > div {
+    margin-bottom: $pt-grid-size;
```

Comment:
That's what it was before, had to change it because I was getting this error:
```
Warning: validateDOMnesting(...): <div> cannot appear as a descendant of <p>. See ... SomeComponent > p > ... > SomeOtherComponent > ReactTooltip > div.
```